desktop/main.js

```diff
@@ -196,6 +196,12 @@ const mainWindow = (() => {
         app.setName('New Expensify');
     }
 
+    /*
+    * Starting from Electron 20, it shall be required to set sandbox option to false explicitly.
+    * Running a preload script contextBridge.js require access to nodeJS modules from the javascript code.
+    * This was not a concern earlier as sandbox used to be false by default for Electron <= 20.
+    * Refer https://www.electronjs.org/docs/latest/tutorial/sandbox#disabling-the-sandbox-for-a-single-process
+    * */
```

Comment:
Please add the comment above the config

```
 contextIsolation: true,
                    
// comment here
sandbox: false,
```

Reply:
@Santhosh-Sellavel that will be like putting big 4 lines of comment in between initialization of browserWindow object which may make hamper the code readability. Hence I preferred. Can you please reconsider your comment.

Reply:
Not a problem

Reply:
I think the comment is fine where it is now. I agree that it's better for readability to keep it above both lines.